### ISSUE TITLE
Really?

### ISSUE THREAD

**Issue by MiaMariPer3** [facebookresearch/Detectron#1045]
I believe you may be doing something that could become unfeasible to adapt to older devices. There are better solutions available since not everyone lives in a bubble, which is why accessibility should be paramount when creating something functional.
